src/Collections/MemoryConfigCollection.php

```diff
@@ -117,9 +115,6 @@ public function checkForDeprecatedConfig($class, $name): void
     {
         $deprecated = $this->getClassConfig('__deprecated', true);
         $data = $deprecated['config'][strtolower($class)][$name] ?? [];
-        if (!empty($data)) {
```

Comment:
Please restore this block, this is not deprecated code, it's code that helps with identifying deprecated code